doc/api/cli.md

```diff
@@ -277,6 +277,12 @@ added: v0.11.15
 Data path for ICU (Intl object) data. Will extend linked-in data when compiled
 with small-icu support.
 
+### `NODE_PRESERVE_SYMLINKS=true`
+<!-- YAML
+added: v6.7.0.
```

Comment:
There should be no `.` here, and generally it’s easiest to simply use `added: REPLACEME` (this likely won’t end up in v6.7.0, which is going to be a security release on Tuesday).